front-end/src/screens/SignupForm/index.tsx

```diff
@@ -171,4 +195,16 @@ export default styled(SignupForm)`
 	i.icon.question.circle:before {
 		color: grey_secondary;
 	}
+
+	.text-muted {
+		display: inline-block;
+		margin-top: 0!important;
+	}
+
+	.ui.form input[type=checkbox]{
+		margin-right: 1rem;
+		vertical-align: middle;
+		height: 2rem;
```

Comment:
This makes it look weird on FF:
![image](https://user-images.githubusercontent.com/33178835/74654484-abed1700-518a-11ea-808f-bc0a059e3a6e.png)
```suggestion
```